File: Python/program/Mahasiswa.py
# class declaration
class Mahasiswa:

    # atribut private
    __nim = ""
    __nama = ""
    __prodi = ""
    __fakultas = ""

   # konstruktor
    def __init__(self, nim="", nama="", prodi="", fakultas=""):
        self.__nim = nim
        self.__nama = nama
        self.__prodi = prodi
        self.__fakultas = fakultas

    # Getter and Setter

    def get_nim(self):
        return self.__nim

    def set_nim(self, nim):
        self.__nim = nim

    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    def get_prodi(self):
        return self.__prodi

    def set_prodi(self, prodi):
        self.__prodi = prodi

    def get_fakultas(self):
        return self.__fakultas

    def set_fakultas(self, fakultas):
        self.__fakultas = fakultas

File: PHP/program/Mahasiswa.php
<?php
// deklarasi kelas mahasiswa
class Mahasiswa {
    // atribut private
    private $nim = '';
    private $nama = '';
    private $prodi = '';
    private $fakultas = '';
    private $foto = '';

    // konstruktor
    public function __construct( $nim = '', $nama = '' , $prodi = '', $fakultas = '',$foto='' ) {
        $this->nim = $nim;
        $this->nama = $nama;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
        $this->foto = $foto;
    }
 /* Getter dan Setter */
    public function set_nim($nim){
        $this->nim = $nim;
    }

    public function get_nim(){
        return $this->nim;
    }
    
    public function set_nama($nama){
        $this->nama = $nama;
    }
    public function get_nama(){
        return $this->nama;
    }
    public function set_prodi($prodi){
        $this->prodi = $prodi;
    }

    public function get_prodi(){
        return $this->prodi;
    }
    
    public function set_fakultas($fakultas){
        $this->fakultas = $fakultas;
    }
    public function get_fakultas(){
        return $this->fakultas;
    }
    public function set_foto($foto){
        $this->foto = $foto;
    }
    public function get_foto(){
        return $this->foto;
    }

}

?>